CoverageProfiler/CoverageProfiler.inputs.json

```diff
@@ -0,0 +1,32 @@
+{
+  "coverageProfile.referenceFasta": "File",
+  "coverageProfile.SamtoolsDepth.samtools_docker": "String (optional, default = \"euformatics/samtools:1.20\")",
+  "coverageProfile.CovProfileViz.cpu": "Int? (optional)",
+  "coverageProfile.DepthOfCoverage.gatk_docker": "String (optional, default = \"broadinstitute/gatk:4.5.0.0\")",
+  "coverageProfile.referenceDict": "File",
+  "coverageProfile.intervals": "File",
+  "coverageProfile.referenceFai": "File",
+  "coverageProfile.alignedBam": "File",
+  "coverageProfile.alignedBamIndex": "File",
+  "coverageProfile.coverageTool": "String (optional, default = \"Samtools\")",
+  "coverageProfile.RegionalDepthProfile.mem_gb": "Int (optional, default = 7)",
+  "coverageProfile.RegionalDepthProfile.disk_size_gb": "Int? (optional, default = 500)",
+  "coverageProfile.DepthOfCoverage.cpu": "Int? (optional)",
+  "coverageProfile.SamtoolsDepth.cpu": "Int? (optional)",
+  "coverageProfile.MinMappingQuality": "Int (optional, default = 20)",
+  "coverageProfile.CovProfileViz.mem_gb": "Int (optional, default = 32)",
+  "coverageProfile.MinBaseQuality": "Int (optional, default = 20)",
+  "coverageProfile.cnv_depth_profile": "Boolean (optional, default = false)",
+  "coverageProfile.CovProfileViz.CovProfileViz_docker": "String (optional, default = \"us-central1-docker.pkg.dev/tag-team-160914/gptag-dockers/covprofileviz:0.0.0\")",
+  "coverageProfile.CovProfileViz.preemptible": "Int? (optional, default = 3)",
+  "coverageProfile.cnvBed": "File? (optional)",
+  "coverageProfile.RegionalDepthProfile.cpu": "Int? (optional)",
+  "coverageProfile.RegionalDepthProfile.preemptible": "Int? (optional, default = 3)",
+  "coverageProfile.CovProfileViz.disk_size_gb": "Int? (optional, default = 500)",
+  "coverageProfile.SamtoolsDepth.mem_gb": "Int? (optional)",
+  "coverageProfile.interval_GCcontent_track": "File",
+  "coverageProfile.RegionalDepthProfile.RegionalDepthProfile_docker": "String (optional, default = \"us-central1-docker.pkg.dev/tag-team-160914/gptag-dockers/covprofileviz:0.0.1\")",
```

Comment:
Is there a dockerfile you used to generate this docker?

Reply:
Yes. It will be in a PR in TAG repo.